packages/roosterjs-editor-dom/lib/clipboard/extractClipboardItems.ts

```diff
@@ -1,131 +1,141 @@
-import readFile from '../utils/readFile';
```

Comment:
Please let me know how can I fix this formatting. I see that `pretty-quick` is part of pre-commit hook but even running manually it didn't fix. Is it because this file hasn't changed since prettier config changed and its doing the right thing?